src/engines/html5/media-source/adapters/native-adapter.js

```diff
@@ -220,10 +220,6 @@ export default class NativeAdapter extends BaseMediaSourceAdapter {
           this._videoElement.src = this._sourceObj.url;
           this._trigger(CustomEventType.ABR_MODE_CHANGED, {mode: this._isProgressivePlayback() ? 'manual' : 'auto'});
         }
-        if (startTime && startTime > -1) {
-          this._videoElement.currentTime = startTime;
-        }
-        this._videoElement.load();
```

Comment:
@dan-ziv how do you account for removing call to `load`? what was the use case for previously having it here?

Reply:
@dan-ziv - so we don't need it here?

Reply:
I don't think so, but lets keep it.
Plz review